coderd/userauth.go

```diff
@@ -261,7 +264,7 @@ func (api *API) userOIDC(rw http.ResponseWriter, r *http.Request) {
 		return
 	}
 	verifiedRaw, ok := claims["email_verified"]
-	if ok {
+	if ok && !api.OIDCConfig.IgnoreEmailVerified {
```

Comment:
nit: maybe add a log entry here to inform that verification failed, but the code will ignore it.